src/domains/sign/network.tf

```diff
@@ -27,3 +39,93 @@ module "io_sign_snet" {
   }
 }
 
+module "io_sign_user_snet" {
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.13.1"
+  name                 = format("%s-user-snet", local.project)
+  resource_group_name  = data.azurerm_virtual_network.vnet_common.resource_group_name
+  virtual_network_name = data.azurerm_virtual_network.vnet_common.name
+  address_prefixes     = ["10.0.103.0/24"]
```

Comment:
where does this value come from?

Reply:
Each FunctionApp needs its own subnet; therefore the subnet belongs to the shared VNET.

The prefix, is just one of the /24 prefixes that was unused at the moment of the creation

Reply:
How do we know that is unused?

Reply:
By looking at the internal list of the used subnets for the VNET, that was not in the list; neither it is a subnet used is some of the peering for that VNET

Reply:
Is there any command that we can launch using the azure cli to obtain this list?

Reply:
`az network vnet subnet list --resource-group $rg_name --vnet-name $vnet_name`